src/core/server/metrics/metrics_service.ts

```diff
@@ -83,5 +92,31 @@ export class MetricsService
       clearInterval(this.collectInterval);
     }
     this.metrics$.complete();
+    this.subscription.unsubscribe();
```

Comment:
@pgayvallet, looking at this `stop` method, we complete `this.metrics$`, which is the source of `metricsObservable`. Isn't that enough to (automatically) close the subscription?

Asking you since you requested to explicitly handle the subscription.